chajda/tsquery/augments.py

```diff
@@ -53,114 +58,225 @@ def augments_gensim(lang, word, config=Config(), n=5):
 
     return words
 
-import os
-import fasttext
-import fasttext.util
-from annoy import AnnoyIndex
+
+#################################################################################
+# fasttext
+#################################################################################
 
 # path used for storing/loading fasttext models and annoy indices
 fasttext_annoy_data_path = os.path.dirname(__file__)
 
+# fasttext_models dictionary will hold the loaded fasttext models as values,
+# and the corresponding langauge as keys
 fasttext_models = {}
+
+
+def load_fasttext(lang):
+    '''
+    Downloads and loads a fasttext model for a particular language,
+    storing the fasttext model object in the fasttext_models dictionary.
+    Due to the size of fasttext models (>1gb), and the loading time,
+    we do not load all models on program startup.
+    Instead, this function is used to load them lazily as needed.
+    lang: language passed from augments_fasttext
+
+    NOTE:
+    The doctest below has been commented out due to requiring more memory than github actions allows for.
+    # >>> load_fasttext_model('ja') or fasttext_models['ja'] != None
+    # True
+    '''
+
+    working_dir = os.getcwd()
+    # suppress_stdout_stderr() is used for redirecting stdout and stderr when downloading and loading fasttext models.
+    # Without this, stdout and stderr from fasttext causes doctests to fail.
+    with suppress_stdout_stderr():
+
+        # fasttext downloads models to the current directory,
+        # but we want to store the model in chaja/tsquery/fasttext,
+        # so we change the working directory before downloading
+        try:
```

Comment:
You can simplify this code  by making `os.mkdir` not fail if the directory already exists; see: https://docs.python.org/3/library/os.html?highlight=exist_ok#os.makedirs

then you don't need the try/except or two calls to `os.chdir`

Reply:
```
working_dir = os.getcwd()
```
should be in this "comment paragraph" so that we know why we're storing this variable